Machine_Learning/Heart_Attack_prediction/Heart_attack_prediction.ipynb

```diff
@@ -0,0 +1,2013 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "80413594",
+   "metadata": {},
+   "source": [
+    "# Exploratory Data Analysis (EDA)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "dbcb807d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "\n",
+    "# Load the dataset\n",
+    "data = pd.read_csv('heart_attack_prediction_dataset.csv')"
```

Comment:
Great work! But please provide either .csv file of the dataset or steps to read the dataset from Kaggle in the README.md file. 
After that will merge the changes.